tests/test_stenciling.py

```
@@ -0,0 +1,191 @@
+        self.Se_intrinsic_thermo = DefectThermodynamics.from_json(
+            f"{self.Se_example_dir}/Se_Intrinsic_Thermo.json.gz"
+        )
+        self.Se_old_new_names_dict = {"vac_1_Se": "v_Se", "Int_Se_1": "Se_i_C2"}
+        self.tight_sm = StructureMatcher(stol=0.02, comparator=ElementComparator(), primitive_cell=False)
+        # TODO: Do one of the Se extrinsic substitutions/interstitials as a test too
+        # TODO: Once tests setup, should look at using `LinearAssignment` in
+        #  `_stencil_target_cell_from_big_cell`; more efficient, robust, and then easier to manage with
+        #  consistency with other parts of the code
+        # TODO: Useful test case could be trying to stencil split vacancies in new supercells...
+
+    def test_Se_20_Å_supercell(self):
```

Ruff (0.11.9)

Function name `test_Se_20_Å_supercell` contains a non-ASCII character

(PLC2401)